Sources/OpenAPIRuntime/Interface/ClientTransport.swift

```diff
@@ -229,10 +229,11 @@ public protocol ClientMiddleware: Sendable {
     ///   - operationID: The identifier of the OpenAPI operation.
     ///   - next: A closure that calls the next middleware, or the transport.
     /// - Returns: An HTTP response.
+    @preconcurrency
```

Comment:
We should not add this and any of the other `@preconcurrency` attributes. Just marking the closure as `@Sendable` seems all we need.